javascript/javascript1/week3/Item array removal.js

```diff
@@ -0,0 +1,24 @@
+const names = [
+ "Peter",
+ "Ahmad",
+ "Yana",
+ "kristina",
+ "Rasmus",
+ "Samuel",
+ "katrine",
+ "Tala",
+];
+const nameToRemove = "Ahmad";
+// Write some code here
+function removeName(name) {
+ if (arguments.length < 1 || typeof name != "string") {
+   return;
+ }
+ if (names.indexOf(name) === -1) {
+   return;
+ }
+ names.splice(names.indexOf(name), 1);
+}
+removeName(nameToRemove);
```

Comment:
Nice work. just a small suggestion from my side and easy way we can write this one with for loop.